crates/telemetry/src/log.rs

```diff
@@ -0,0 +1,18 @@
+use std::sync::OnceLock;
+
+use crate::env;
+
+/// Takes a Spin application log and emits it as a tracing event. This acts as a compatibility layer
+/// to easily get Spin app logs as events∆1 in our OTel traces.
+pub fn app_log_to_tracing_event(buf: &[u8]) {
+    static CELL: OnceLock<bool> = OnceLock::new();
```

Comment:
Feels like you could move this into the `app_log_to_tracing_event_disabled` function and avoid all the CELL clutter here?

Reply:
Fair. I've been treating `env.rs` as purely a retrieval layer and so in my mind the caching makes sense to live here. I could be convinced otherwise.